packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -1829,14 +1830,14 @@ test('mutable liquidity triggers and interest', async t => {
     await E(bobVault).getCurrentDebt(),
   );
   // 5 days pass
-  manualTimer.tick();
-  manualTimer.tick();
-  manualTimer.tick();
-  manualTimer.tick();
+  await manualTimer.tick();
+  await manualTimer.tick();
+  await manualTimer.tick();
+  await manualTimer.tick();
   await manualTimer.tick();
   await waitForPromisesToSettle();
 
-  shortfallBalance += 42n;
+  shortfallBalance += 46n;
```

Comment:
I'd like to as well but am balancing that against other priorities. Are you requesting that I figure out what happened and report here before this can be approved?

Reply:
I do consider that a blocking issue. I can probably make time to investigate in the next few days if you want to assign the PR to me.

Reply:
Ok, I'll assign to you and pull it back if I find time.